File: api/src/main/java/com/mystore/manager/api/admin/payload/SessionLogPayload.java
package com.mystore.manager.api.admin.payload;

import com.mystore.manager.api.common.payload.GlobalDatePayload;
import lombok.Data;
import lombok.EqualsAndHashCode;

@Data
@EqualsAndHashCode(callSuper = true)
public class SessionLogPayload extends GlobalDatePayload {
    private Integer userId;
    private String userUsername;
    private String userFullName;
    private Integer posId;
    private String posCode;
    private String posName;
    private String ipAddress;
    private String loginAt;
}

File: ui-admin/src/app/backend/payloads/admin/session-log-payload.ts
export class SessionLogPayload {
    id: number;
    userId: number;
    userUsername: string;
    userFullName: string;
    posId: number;
    posCode: string;
    posName: string;
    ipAddress: string;
    loginAt: string;
    createdAt: string;

    constructor() {
        this.id = null;
        this.userId = null;
        this.userUsername = null;
        this.userFullName = null;
        this.posId = null;
        this.posCode = null;
        this.posName = null;
        this.ipAddress = null;
        this.loginAt = null;
        this.createdAt = null;
    }
}
